packages/shared/src/utils/cfd/cfd.ts

```diff
@@ -39,7 +44,7 @@ type TGetCFDAccountKey = TGetAccount & {
 // *
 // sub_account_type financial_stp only happens in "financial" market_type
 export const getCFDAccountKey = ({ market_type, sub_account_type, platform, shortcode }: TGetCFDAccountKey) => {
-    if (market_type === 'all') {
+    if (market_type === 'all' && platform !== CFD_PLATFORMS.MT5) {
```

Comment:
Help to update the comment above, it is outdated

Reply:
ok